server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -493,6 +493,8 @@ public void apply(Settings value, Settings current, Settings previous) {
                 IndicesFieldDataCache.INDICES_FIELDDATA_CACHE_SIZE_KEY,
                 IndicesRequestCache.INDICES_CACHE_QUERY_SIZE,
                 IndicesRequestCache.INDICES_CACHE_QUERY_EXPIRE,
+                IndicesRequestCache.INDICES_REQUEST_CACHE_CLEANUP_INTERVAL_SETTING,
+                IndicesRequestCache.INDICES_REQUEST_CACHE_STALENESS_THRESHOLD_SETTING,
```

Comment:
Can you also create documentation issue/PR for these new settings

Reply:
@sgup432 has an issue open to track this, itseems he will have a PR open soon including these settings 

https://github.com/opensearch-project/documentation-website/issues/6300